client/app/bundles/course/survey/pages/ResponseIndex/index.jsx

```diff
@@ -92,8 +94,18 @@ const translations = defineMessages({
   },
 });
 
-class ResponseIndex extends Component {
-  static computeStatuses(responses) {
+const ResponseIndex = (props) => {
+  const { dispatch, survey, responses, isLoading } = props;
+  const { palette } = useTheme();
+  const [state, setState] = useState({
+    includePhantomsInStats: false,
+  });
+
+  useEffect(() => {
+    dispatch(fetchResponses());
+  }, []);
```

Comment:
add dispatch as dependency. Please read: https://overreacted.io/a-complete-guide-to-useeffect/